time stamp
float32 desired_rate
float32 current_rate

sensor_msgs/JointState joint_state